mmcv/ops/deform_conv.py

```diff
@@ -256,7 +256,7 @@ def __init__(self,
         self.reset_parameters()
 
     def reset_parameters(self):
-        n = self.in_channels
+        n = self.in_channels // self.groups # fu added
```

Comment:
It would be better to refer to this PR in the comments above this line.

Reply:
Maybe we can invoke `nn.init.kaiming_uniform_` to initialize the weight here, which will make the initialization more clear